lld/COFF/Chunks.cpp

```diff
@@ -946,7 +946,7 @@ void ECCodeMapChunk::writeTo(uint8_t *buf) const {
   auto table = reinterpret_cast<chpe_range_entry *>(buf);
   for (uint32_t i = 0; i < map.size(); i++) {
     const ECCodeMapEntry &entry = map[i];
-    uint32_t start = entry.first->getRVA();
+    uint32_t start = entry.first->getRVA() & ~0xfff;
```

Comment:
This looks reasonable, but do we use explicit `0xfff` for page alignment here/elsewhere so far, or do we have some constant for that?

Reply:
Yes, we already use `0xfff` in several places in LLD, mostly, but not exclusively, in `Chunks.cpp`.